eng/pipelines/docindex.yml

```diff
@@ -33,14 +33,24 @@ jobs:
             - Name: $(DocRepoOwner)/$(DocRepoName)
               WorkingDirectory: $(DocRepoLocation)
 
+      # Set $(DefaultBranch) for use in condition on "Update Docs Onboarding"
```

Comment:
This should no longer be needed.

Reply:
Plus it is also being set below.

Reply:
It pulls default branch on different repos.

You are right, the step is used by the condition check. Since we removed the condition check, It is supposed to be removed.